localization/en.ftl

```diff
@@ -220,3 +220,29 @@ multiline-opt-automaticallyMultilinify-desc = If enabled, automatically split ex
 multiline-opt-automaticallyMultilinify-name = Automatically Multilinify
 multiline-opt-multilinifyDelayAfterEdit-desc = Multiline expressions should be updated after no edits are made for this number of milliseconds.
 multiline-opt-multilinifyDelayAfterEdit-name = Edit Delay (ms)
+
+## Custom MathQuill Config
+custom-mathquill-config-name = Custom MathQuill Config
+custom-mathquill-config-desc = Lets you edit MathQuill config options to change how equation input works.
```

Comment:
```suggestion
custom-mathquill-config-desc = Changes how equation inputs work.
```

Started rewriting this to put a warning about 'More Greek Letters,' but decided it wasn't worth the effort. The shorter description could still be nice, up to you.